### ISSUE TITLE
[sommelier] fails to start  architecture:x86_64

### ISSUE THREAD

**Issue by sparklescrewdriver** [chromebrew/chromebrew#8404]
To try and resolve this issue I have tried the solution in issue #5646 this failed however here is a copy of what my log is saying 
'xset:  unable to open display ":0"
xset:  unable to open display ":0"
sommelier -X     --x-display=:0     --scale=1.0     --direct-scale     --glamor     --force-drm-device=/dev/dri/renderD128     --display=wayland-0     --xwayland-path=/usr/local/bin/Xwayland     --xwayland-gl-driver-path=/usr/local/lib64/dri     --peer-cmd-prefix=/usr/local/lib64/ld-linux-x86-64.so.2     --noop-driver      --no-exit-with-child     /bin/sh -c xauth generate :0 . trusted
error: Kernel is too old (4.16+ required) or unusable for Iris.
Check your dmesg logs for loading failures.

error: Kernel is too old (4.16+ required) or unusable for Iris.
Check your dmesg logs for loading failures.

_XSERVTransmkdir: ERROR: euid != 0,directory /tmp/.X11-unix will not be created.
error: Kernel is too old (4.16+ required) or unusable for Iris.
Check your dmesg logs for loading failures.

error: Kernel is too old (4.16+ required) or unusable for Iris.
Check your dmesg logs for loading failures.
Refusing to try glamor on llvmpipe
EGL setup failed, disabling glamor
Failed to initialize glamor, falling back to sw
xauth:  file /home/chronos/.Xauthority does not exist
The XKEYBOARD keymap compiler (xkbcomp) reports:
 Warning:          Unsupported maximum keycode 569, clipping.
                 X11 cannot support keycodes above 255.'

it aslo says this when i try to use sommelier  

`some sommelier processes failed to start
sommelier processes running:  
 5187 /usr/local/bin/../lib64/ld-linux-x86-64.so.2 --library-path /usr/local/bin/../lib64 --inhibit-rpath  /usr/local/bin/sommelier.elf -X --x-display=:0 --scale=1.0 --direct-scale --glamor --force-drm-device=/dev/dri/renderD128 --display=wayland-0 --xwayland-path=/usr/local/bin/Xwayland --xwayland-gl-driver-path=/usr/local/lib64/dri --peer-cmd-prefix=/usr/local/lib64/ld-linux-x86-64.so.2 --noop-driver --no-exit-with-child /bin/sh -c xauth generate :0 . trusted
8169 /usr/local/bin/../lib64/ld-linux-x86-64.so.2 --library-path /usr/local/bin/../lib64 --inhibit-rpath  /usr/local/bin/sommelier.elf -X --x-display=:0 --scale=1.0 --direct-scale --glamor --force-drm-device=/dev/dri/renderD128 --display=wayland-0 --xwayland-path=/usr/local/bin/Xwayland --xwayland-gl-driver-path=/usr/local/lib64/dri --peer-cmd-prefix=/usr/local/lib64/ld-linux-x86-64.so.2 --noop-driver --no-exit-with-child /bin/sh -c xauth generate :0 . trusted
13253 /usr/local/bin/../lib64/ld-linux-x86-64.so.2 --library-path /usr/local/bin/../lib64 --inhibit-rpath  /usr/local/bin/sommelier.elf -X --x-display=:0 --scale=1.0 --direct-scale --glamor --force-drm-device=/dev/dri/renderD128 --display=wayland-0 --xwayland-path=/usr/local/bin/Xwayland --xwayland-gl-driver-path=/usr/local/lib64/dri --peer-cmd-prefix=/usr/local/lib64/ld-linux-x86-64.so.2 --noop-driver --no-exit-with-child /bin/sh -c xauth generate :0 . trusted`

on chromeos Version 114.0.5735.119 (Official Build) (64-bit)

**Comment by satmandu** [comment]
What does `uname -a` give?

**Comment by sparklescrewdriver** [comment]
Linux localhost 4.14.313-20323-gc32c2d54e434

**Comment by satmandu** [comment]
I'm going to have to replicate this locally... I do have a machine with a `4.14` kernel...

**Comment by satmandu** [comment]
This is what I see:
```
crosh> shell
Sommelier can use direct scaling.
Sommelier SCALE is set to 1.0.
SCALE may be manually set in ~/.sommelier.env .
sommelier processes running:  6831
chronos@localhost / $ uname -a
Linux localhost 4.14.313-20323-gc32c2d54e434 #1 SMP PREEMPT Sun Jun 4 19:13:07 PDT 2023 x86_64 Intel(R) Celeron(R) N4000 CPU @ 1.10GHz GenuineIntel GNU/Linux
chronos@localhost / $ glxinfo -B
name of display: :0
display: :0  screen: 0
direct rendering: Yes
Extended renderer info (GLX_MESA_query_renderer):
    Vendor: Mesa (0xffffffff)
    Device: llvmpipe (LLVM 16.0.5, 128 bits) (0xffffffff)
    Version: 23.1.2
    Accelerated: no
    Video memory: 3805MB
    Unified memory: yes
    Preferred profile: core (0x1)
    Max core profile version: 4.5
    Max compat profile version: 4.5
    Max GLES1 profile version: 1.1
    Max GLES[23] profile version: 3.2
Memory info (GL_ATI_meminfo):
    VBO free memory - total: 31 MB, largest block: 31 MB
    VBO free aux. memory - total: 247 MB, largest block: 247 MB
    Texture free memory - total: 31 MB, largest block: 31 MB
    Texture free aux. memory - total: 247 MB, largest block: 247 MB
    Renderbuffer free memory - total: 31 MB, largest block: 31 MB
    Renderbuffer free aux. memory - total: 247 MB, largest block: 247 MB
Memory info (GL_NVX_gpu_memory_info):
    Dedicated video memory: 4294314663 MB
    Total available memory: 4294318469 MB
    Currently available dedicated video memory: 31 MB
OpenGL vendor string: Mesa
OpenGL renderer string: llvmpipe (LLVM 16.0.5, 128 bits)
OpenGL core profile version string: 4.5 (Core Profile) Mesa 23.1.2 (git-ffdc2d248d)
OpenGL core profile shading language version string: 4.50
OpenGL core profile context flags: (none)
OpenGL core profile profile mask: core profile

OpenGL version string: 4.5 (Compatibility Profile) Mesa 23.1.2 (git-ffdc2d248d)
OpenGL shading language version string: 4.50
OpenGL context flags: (none)
OpenGL profile mask: compatibility profile

OpenGL ES profile version string: OpenGL ES 3.2 Mesa 23.1.2 (git-ffdc2d248d)
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 3.20

chronos@localhost / $ 
```
Can you try a reinstall of Chromebrew?
```
curl -Ls git.io/vddgY | bash
crew reinstall sommelier
```

**Comment by sparklescrewdriver** [comment]
Reinstalling fixed it
